web/src/lib/utils/thumbnail-util.ts

```diff
@@ -40,24 +41,33 @@ export function getAltText(asset: AssetResponseDto) {
     return asset.exifInfo.description;
   }
 
-  let altText = 'Image taken';
-  if (asset.exifInfo?.city && asset.exifInfo.country) {
-    altText += ` in ${asset.exifInfo.city}, ${asset.exifInfo.country}`;
+  const $t = getFormatterSync();
```

Comment:
Can this be a derived store instead?

Reply:
This util function isn't needed if tests are being run against the actual translations - I've removed it.

Reply:
Thanks for the changes. This function should still use a derived to preserve reactivity when the language changes:

```ts
const getAltText = derived(t, ($t) => {
  return (asset: AssetResponseDto) => {
    
  };
});
```

Reply:
Ok, I can make that change. I'm not sure how to test this - what is the scenario that this derived statement is needed for? I'm seeing the alt text changing when I go to the settings and back.

Reply:
And thanks for the feedback! This is good to know going forward.

Reply:
This scenario would occur when the language changes while you're on a page where `getAltText` is used. I don't think it's currently an issue, because going to settings and back will cause a re-render.